docs/github_actions.rst

```diff
@@ -441,6 +498,233 @@ gpu-cuda (Clang14Dev-MPI-CUDA-AFQMC-Offload-Real)
 +---------------+-------------------------------------------------------------------------------------------------------------------------------------+
 
 
+gpu-cuda (Intel19-MPI-CUDA-AFQMC-Real-Mixed)
```

Comment:
Please add a comment or note that for Intel builds, the environment script that ship with the Intel compiler needs to be sourced (e.g. compilervars.sh).

Reply:
These issues should be resolved, can you confirm? @williamfgc

Reply:
Ping @williamfgc